inc/refusedequipment.class.php

```diff
@@ -238,4 +240,38 @@ public function isDynamic() {
    static function canPurge() {
       return static::canUpdate();
    }
+
+   /**
+    * Handle inventory request, and returns redirection url
+    *
+    * @return string
+    */
+   public function handleInventoryRequest(Request $request) {
+      $status = $request->getInventoryStatus();
+
+      if (!isset($status['itemtype'])) {
+         //no itemtype found, redirect to refused logs - should not happen
```

Comment:
Maybe a message should be added. Without a message, user will not understand why nothing is done.

Reply:
I've dropped this part, this should not happen